snippets/mapped-table.md

```diff
@@ -0,0 +1,45 @@
+### MappedTable
+
+Renders a table with rows dynamically created from an array of data.
+
+Use the value of the `headers` prop to conditionally render a table with a header. Use `Array.prototype.map` to render every item in `data` as a `<tr>` element and give it a `key`. `data` can either be an array of objects with the `id` and `value` properties or an array of primitives. Omit the `headers` prop to render a `table` without headers.
+
+```jsx
+function MappedTable(props) {
```

Comment:
A lot of people pass props like this:

```js
function MappedTable({ headers, data }) {
```

They destructure the props in the function definition (or functional component definition). This is serves kinda as a documentation since you see all the props that need to be passed to that function/component :D

Reply:
Makes total sense to me. Much more readable in every aspect 👍 

@Chalarangelo @skatcat31 could you guys give your opinion on this one? I think we should enforce this rule.  I vote +1 for this for the following reasons:

- Very readable
- Users don't need to look into component's code in order to see what props are used
- Eliminates looking into examples of obvious components/props 👍 

Thanks @nikolalsvk

Reply:
I agree, this is a good idea and will help a lot in the long run.